myPortfolio/index.html

```diff
@@ -1,25 +1,85 @@
-    </div> 
-
-    <div class="containers">
-        <h1>Whitney-Rene</h1>
-        <h2>Aspiring Software Engineer</h2>
-        <p>Welcome to my portfolio page. ...</p>
-    </div>
+    <title>Whitney-Rene's Portfolio Page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <div class="photoAndBio">
+
+      <div class="containers">
+          <img id="photoOfMe" src="./Whitney-Rene-Chambers-2023-h2.png" alt="Photo of Whitney-Rene">
+      </div> 
+
+      <div class="containers" id="Bio">
+        <div class="headings">
+          <h1>Whitney-Rene</h1>
+          <h2>Software Engineer</h2>
+        </div>
+          <p id="paraFont">Thank you for visiting my portfolio page.  Again, my name is Whitney-Rene, and I am an aspiring software engineer.  I am currently a part of a 6 month with an organization called Techtonica.  Techtonica's mission is to bridge the gap between women of color and the tech industry.  I could not be happier to be a participant in the 2023 H2 Co-Hort. In my free time, I enjoy being outside, connecting with friends and painting my nails!  Please feel free to browse around my webpage.  My contact is listed below.  And please fill out the form below.  I would love to connect! </p>
+      </div>
     
-  </div>  
+      <div class="Projects">
+        
+        <h2 id="projectTitle">Projects</h2>
+        
+        <ul>
+          <li><a href="https://github.com/Whitney-Rene/techtonica-assignments">Current Projects</a></li>
+          
+          <li><a href="https://github.com/Whitney-Rene/JS-exercises">JS-Exercises</a></li>
+        </ul>
+      
+      </div>
+
+    </div>  
+
+
+
+
+      <div class="ContactInfoAndForm">
+
+        <h2 id="contactTitle">Contact Me!</h2>
+        <h5 id="h5One">Contact me via email: whhtnchambers76@gmail.com</h5>
+        <h5 id="h5Two"><a id="specialtag" href="https://www.linkedin.com/in/whitney-rene-chambers/">Connect with me on LinkedIn</a></h5>
+        
+        <form>
+
+          <label for="fnameinput">What is your first name?</label>
+          <input id="fnameinput" type="text" name="firstname" placeholder="enter first name" required>
+        
+          <br>
+
+          <label for="email">What is your email?</label>
+          <input id="email" type="text" name="email" placeholder="enter email" required>
+          
+          <br>
+          
+          <label for="number">Would you like to share your number?</label>
+          <input id="number" type="tel" name="phoneNumber" placeholder="enter number (optional)">
+          
+          <br>
+          
+          <label for="address">What is your address?</label>
+          <input id="address" type="text" name="address" placeholder="enter address (optional)">
+
```

Comment:
Consider implementing another semantic HTML tag here, `<fieldset>`, to wrap all the fields.